nautilus_trader/cache/cache.pyx

```diff
@@ -2072,17 +2087,25 @@ cdef class Cache(CacheFacade):
             str base_quote
             Price bid
             Price ask
+            Bar bid_bar
+            Bar ask_bar
         for instrument_id, base_quote in self._xrate_symbols.items():
             if instrument_id.venue != venue:
                 continue
 
             ticks = self._quote_ticks.get(instrument_id)
-            if not ticks:
+            if ticks:
+                bid = ticks[0].bid
+                ask = ticks[0].ask
+            else:
                 # No quotes for instrument_id
-                continue
+                bid_bar = self._bars_for_instrument_id[PriceType.BID].get(instrument_id)
+                ask_bar = self._bars_for_instrument_id[PriceType.ASK].get(instrument_id)
+                if not bid_bar or not ask_bar:
```

Comment:
Cython will work more efficiently with the full `is None` rather than Python object truthiness

I can tidy this up though